internal/cli/cli.go

```diff
@@ -134,3 +144,26 @@ func collectionDocuments(serverUrl string, apiKey string, ids []string) error {
 	}
 	return nil
 }
+
+func collectionList(serverUrl string, apiKey string) error {
+	oc := outline.New(serverUrl, &http.Client{}, apiKey)
+	var collections []*outline.Collection
+	err := oc.Collections().List().Do(context.Background(), func(c *outline.Collection, err error) (bool, error) {
+		if err != nil {
+			// We are going to ignore errors for now...
+			return true, nil
```

Comment:
Already 👌. But this would also mean that in case 10 collections exist, but there is an issue at collection 5, we stop and ignoring the other/last (maybe working) 5 collections? Correct?

Reply:
yes that is correct! However it is highly unlikely that the issue would be with one collection. The issue would most likely be while fetching a whole page. If we already started receiving and printing some items then the possibility of a unrecoverable failure say a 4XX becomes very unlikely. The 5XX or timeouts are still quite likely on the other hand. But luckily we can deal with these at the HTTP client level itself.